source/operations/concepts/scanner.rst

```diff
@@ -0,0 +1,95 @@
+At the cluster level, the scanner splits all buckets into sets and scans one set at a time.
+The scanner starts with new buckets, then randomizes the scanning of other buckets.
+The scanner completes checks on all bucket sets before starting over with a new set of scans.
+
+At the bucket level, the scanner groups items in buckets and scans selected items from that bucket.
+The scanner selects objects for a scan based on a hash of the object name.
+Over a span of 16 scans, MinIO checks every object it knows about.
+
+Scan Length
+-----------
+
+Multiple factors impact the time it takes for a scan to complete.
+
+Some of these factors include:
+- Type of hard drives provided to MinIO
+- Throughput available
+- Number and size of objects
+- Other activity on the MinIO Server
+
+For example, MinIO may pause the scanner to make I/O operations available for read and write requests.
+This lengthens the time it takes for a scan to complete.
+
+MinIO waits between each scan by a factor multiplication of the time it takes each scan operation to complete.
+By default, the value of this factor is ``10.0``, meaning MinIO waits 10x the length of an operation after one scan completes before starting the next scan.
+MinIO also applies a maximum wait time between operations, set to ``15s`` by default.
+
+Scanner Performance
+-------------------
+
+Scanner performance typically depends on the available node resources, the size of the cluster, the number of erasure sets compared to the number of drives, and the complexity of bucket hierarchy (objects and prefixes).
+For example, a cluster that starts with 100TB of data that then grows to 200TB of data may require more time to scan the entire namespace of buckets and objects given the same hardware and workload.
+Likewise, a single erasure set of 16 drives takes longer to scan than the same number of drives split into two erasure sets of 8 drives each.
+
+MinIO treats the scanner as a background task and pauses it in favor of completing read and write requests on the cluster.
+As the cluster or workload increases, scanner performance decreases as it yields more frequently to ensure priority of normal S3 operations.
+
+.. include:: /includes/common/scanner.rst
+   :start-after: start-scanner-speed-config
+   :end-before: end-scanner-speed-config
+
+Scanner Metrics
+---------------
+
+MinIO provides a number of `metrics related to the scanner <https://min.io/docs/minio/linux/operations/monitoring/metrics-and-alerts.html#scanner-metrics>`__.
+
+The metrics updated by the scanner, including usage metrics, reflect the last completed scan. 
+``PUT`` or ``DELETE`` operations since the last scan do not update in the usage until the next scan of the affected bucket(s).
+
+Use ``mc admin scanner info`` to see the current status of the scanner and the time since the last full scan.
+This can help in understanding the metrics provided by the scanner operation.
```

Comment:
Maybe rearrange this a bit? In part to sidestep some technicalities around time series metrics that aren't, strictly speaking, "updated." (You get new values, at a new future point in time.)

```suggestion
Use ``mc admin scanner info`` to see the current status of the scanner and the time since the last full scan.
This can help in understanding the metrics provided by the scanner operation.

Scanner metrics, including usage metrics, reflect the last completed scan.
``PUT`` or ``DELETE`` operations since the last scan do not update in the usage until the next scan of the affected bucket(s).
```